docs/user-guide/src/bmo/raid.md

```diff
@@ -0,0 +1,169 @@
+# RAID setup
+
+[RAID](https://en.wikipedia.org/wiki/RAID) is a technology that allows creating
+volumes with certain properties out of 2 or more physical disks. Depending on
+the [RAID level](https://en.wikipedia.org/wiki/Standard_RAID_levels), you may
+be able to merge several disks into a larger one or achieve redundancy.
+
+Metal3 supports two RAID implementation:
+
+- *Hardware RAID* is implemented by hardware itself and can be configured
+  through the machine's BMC.
+- *Software RAID* is implemented by the Linux kernel and can be configured
+  using the standard `mdadm` tool.
+
+To create or delete RAID volumes, you need to edit the `spec.raid` field of the
+`BareMetalHost` resource, changing either the `hardwareRAIDVolumes` or the
```

Comment:
Do we support the softwareRAIDVolumes bit? I remember only testing via hardwareRAIDVolumes myself.

Reply:
We don't support it in OpenShift, but it exists upstream in Metal3.

Reply:
Perhaps we can also mention a little early in the document that software RAID support is experimental.